### PR TITLE
Added support for oci_set_call_timeout function

### PR DESCRIPTION
**Pull request by ognjenVlad**
I am submitting PR as requested https://github.com/yajra/laravel-oci8/issues/621. Support for oci_set_call_timeout(https://www.php.net/manual/en/function.oci-set-call-timout.php) is added as requested.

### CODE REVIEW

File: src/Pdo/Oci8.php [hunk 1]
```diff
@@ -479,6 +479,21 @@ private function connect($dsn, $username, $password, array $options, $charset)
         }
     }
 
+    /**
+     * Sets a timeout limiting the maxium time a database round-trip using this connection may take
+     *
+     * @param int $time_out
+     * @return bool
+     */
+    private function setCallTimeout($time_out)
```
Comment (yajra) [inline src/Pdo/Oci8.php:488]: Looks good but I think this should be a ` public` method. Thanks!

**Review by yajra — CHANGES_REQUESTED** [review]

File: src/Pdo/Oci8.php (hunk 1 above)
Comment (ognjenVlad) [reply src/Pdo/Oci8.php:488]: Yeah, sorry, my bad, I will push changes in a moment

**Comment by yajra** [comment]
Released on v2.2.0 🚀 Thanks!

**Comment by ognjenVlad** [comment]
> Released on v2.2.0 🚀 Thanks!

Thank you! That was fast!

**Comment by mbackonja** [comment]
@yajra It would be nice if you can merge this also to v3 (for PHP 8 support)

**Pull request merged**